libs/libskynet/src-test/seed.test.ts

```diff
@@ -41,3 +46,50 @@ test("generateSeedPhraseDeterministic", () => {
     expect(found).toBe(true);
   }
 });
+
+// TestMyskyEquivalence is a test that checks that the way libskynet derives
+// the mysky seed for a user matches the code that derived a mysky seed for a
+// user in skynet-mysky. Following the test are some unique dependencies so
+// that the simulated mysky derivation is as close to the original code as
+// possible.
+import nacl from "tweetnacl";
```

Comment:
This import is specific to **only** this test

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#description

> import declarations are only permitted at the top-level of modules

While declaring import in a module body might work now, probably because it's being transformed to commonjs on the fly (?) it seems to be against the spec and honestly looks off just by looking at it. I would just move it to top of the file to conform to the spec and make sure future changes in our build or test pipelines will not break the code.

Reply:
The spec says 'top level' which is not the same as 'top of file'. "top level" means you can't declare an import inside of a function or restricted scope. This code is fully conformant to the spec.